x-pack/filebeat/input/httpjson/value_tpl.go

```diff
@@ -362,6 +363,11 @@ func hashStrings(typ string, data []string) []byte {
 	return h.Sum(nil)
 }
 
+func hexDecode(enc string) string {
+	decodedString, _ := hex.DecodeString(enc)
```

Comment:
The ignoring of the error troubles me; the `hex` decode functions return the progress state up to the error rather than an empty slice. I would suggest conditionally returning `""` if the decode fails.

Reply:
Updated it to conditionally return empty string in case of error.